packages/workflow/src/Interfaces.ts

```diff
@@ -1598,6 +1598,11 @@ export interface INodeType {
 			[method in WebhookSetupMethodNames]: (this: IHookFunctions) => Promise<boolean>;
 		};
 	};
+	customOperations?: {
```

Comment:
This introduces entirely new field to the Node Type interface. Can we have documentation what this is and what it should be used for